bokeh/models/ranges.py

```diff
@@ -90,6 +100,19 @@ class DataRange1d(DataRange):
     automatically computed end value.
     """)
 
+    bound_lower = Float(help="""
+    The minimum value that the range is allowed to go to - typically used to prevent
+    the user from panning/zooming/etc away from the data. If not provided, will
+    default to the min of the data.
+    """)
```

Comment:
> There's no sense which "no min" make sense to me, e.g. (there's always a min, sometimes it is computed), so I would not apply this suggestion to min

Hi @bryevdv, I'm confused. can you explain again?

To my mind: `DataRange1d` `start` looks like this:

````python
    start = Float(help="")
````
If you provide no start then the value is computed.

Similarly, I am proposing:
````python
    bound_lower = Float(help="")
````
and if you provide no bound_lower then the value is computed.

You said:
> I would suggest None means no bound, "auto" means default to the min of the data, and explicit values are explicit bounds. So this would be Either(Auto, Float)

Why does `None` mean "no bound" in the case of `bound_lower` but "please compute" in the case of `start`?